test/bench/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bench",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "main": "node lib/index.js",
+    "build": "npx tsc",
+    "format": "npx eslint --ext '.ts' src --fix"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/node": "^18.16.3",
+    "typescript": "^5.0.4",
+    "eslint": "^8.39.0",
+    "prettier": "^2.8.8",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-prettier": "^4.2.1"
+  },
+  "dependencies": {
+    "bson": "file:../.."
```

Comment:
from team chat, let's try and reuse the set up in this repo for compiling and linting